pagespeed/system/redis_cache.cc

```diff
@@ -599,6 +619,18 @@ bool RedisCache::Connection::EnsureConnection() {
   return state_ == kConnected;
 }
 
+bool RedisCache::Connection::EnsureDatabaseSelection() {
+  RedisReply reply = RedisCommand(StrCat("SELECT ",
```

Comment:
If this is incompatible with clustering, then maybe we should conditionalize this on options->has_database_index().

Even if it's not incompatible with clustering, should we just omit the explicit db index selection if not specified in the config?  I'm not enough of a redis expert to know the right thing here, but if it's working today maybe we shouldn't bother sending the command unless explicitly configured.

Reply:
Added method has_redis_database_index() to check property from config. Selecting database only if property present in config.